bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -261,10 +261,10 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
 
   CONFIG_PARAM(enableEventGroups, bool, false, "A flag to specify whether event groups are enabled or not.");
 
-  CONFIG_PARAM(enablePreProcessorMemoryPool,
-               bool,
-               true,
-               "A flag to specify whether to use a memory pool in PreProcessor or not");
+  CONFIG_PARAM(preProcessorMemoryPoolMode,
+               uint32_t,
+               1,
+               "0: do not user a memory pool in PreProcessor, 1: use multi size buffer pool, 2: use raw memory pool");
```

Comment:
Did you mean "0: do not use"?

Reply:
Yes, strange - I fixed it myself. probably forgot to commit.
Thanks